File: src/app/shared/models/player.ts
export interface Player {
  id: number;
  name: string;
  lastName: string;
  catchPhrase: string;
  penalities: number;
  isOut: boolean;
  image?: string;
  achievementsId?: number[];
  nasScore?: number;
}

export interface PlayersAchievementsRef {
  id: number;
  achievementId: number;
  playerId: number;
}

File: nazdrovia/lib/shared/models/player.dart
import 'package:flutter/widgets.dart';

class Player {
  int id;
  String name;
  String lastName;
  String catchPhrase;
  int penalities;
  bool isOut;
  String image;
  List<int> achievementsId;
  String nasScore;
}

class PlayersAchievementsRef {
  int id;
  int achievementId;
  int playerId;
}
